modules/client/src/internal/client/methods.ts

```diff
@@ -761,13 +761,12 @@ export class ClientMethods extends ClientCore implements IClientMethods {
   /**
    * Retrieves metadata for DAO with given identifier (address or ens domain)
    *
-   * @param {DaoQueryParams} {
+   * @param {DaoQueryParams} params
    *     limit = 10,
```

Comment:
How does this render?

Reply:
type = DaoQueryParams
name = parms
description = following (limit=10, etc.)

Reply:
Added a - in front of each description argument so it renders as a list.